docs/api-diffs/9.0/rc.1/diff.md

```diff
@@ -0,0 +1,1290 @@
++            public static IResourceBuilder<TDestination> WithReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IConstructResource<Topic>> topic, string? configSection = null) where TDestination : IResourceWithEnvironment;
++        }
++        public static class SQSResourceExtensions {
++            public static IResourceBuilder<IConstructResource<Queue>> AddSQSQueue(this IResourceBuilder<IStackResource> builder, [ResourceNameAttribute] string name, IQueueProps? props = null);
++            public static IResourceBuilder<TDestination> WithReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IConstructResource<Queue>> queue, string? configSection = null) where TDestination : IResourceWithEnvironment;
++        }
+     }
+     namespace Aspire.Hosting.AWS {
+-        public class AWSProvisioningException : Exception {
+-            public AWSProvisioningException(string message, Exception? innerException = null);
+-        }
++        public interface IAWSResource : IResource {
++            IAWSSDKConfig AWSSDKConfig { get; set; }
++            TaskCompletionSource ProvisioningTaskCompletionSource { get; set; }
++        }
+     }
++    namespace Aspire.Hosting.AWS.CDK {
++        public delegate T ConstructBuilderDelegate<out T>(Construct scope) where T : IConstruct;
++        public delegate string ConstructOutputDelegate<in T>(T construct) where T : IConstruct;
++        public interface IConstructResource : IResource, IResourceWithConstruct, IResourceWithParent, IResourceWithParent<IResourceWithConstruct>
++        public interface IConstructResource<out T> : IConstructResource, IResource, IResourceWithConstruct, IResourceWithConstruct<T>, IResourceWithParent, IResourceWithParent<IResourceWithConstruct> where T : IConstruct
++        public interface IResourceWithConstruct : IResource {
++            IConstruct Construct { get; }
++        }
++        public interface IResourceWithConstruct<out T> : IResource, IResourceWithConstruct where T : IConstruct {
++            new T Construct { get; }
++        }
++        public interface IStackResource : IAWSResource, ICloudFormationResource, ICloudFormationTemplateResource, IResource, IResourceWithConstruct {
++            Stack Stack { get; }
++        }
++        public interface IStackResource<out T> : IAWSResource, ICloudFormationResource, ICloudFormationTemplateResource, IResource, IResourceWithConstruct, IResourceWithConstruct<T>, IStackResource where T : Stack {
++            new T Stack { get; }
++        }
++    }
+     namespace Aspire.Hosting.AWS.CloudFormation {
+-        public interface ICloudFormationResource : IResource {
++        public interface ICloudFormationResource : IAWSResource, IResource {
+-            IAWSSDKConfig AWSSDKConfig { get; set; }
+-            TaskCompletionSource ProvisioningTaskCompletionSource { get; set; }
++            string StackName { get; }
```

Comment:
Breaking change

Reply:
what is the replacement for removed API or suggestion?